tools/gather_logs_docker.sh

```diff
@@ -0,0 +1,23 @@
+rm -rf ovn-control-plane
+rm -rf ovn-worker
+rm -rf ovn-worker2
+mkdir -p ovn-control-plane
+mkdir -p ovn-worker
+mkdir -p ovn-worker2
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump.pcap .
+docker cp ovn-control-plane:/var/log/openvswitch/tcpdump_logs.log .
+kubectl --kubeconfig=/home/alexey/admin.conf get pods --all-namespaces -o wide > kube-topology
```

Comment:
could you please remove references to alexey ;-)

Reply:
reference  removed